docs/src/test/scala/docs/http/scaladsl/server/directives/MarshallingDirectivesExamplesSpec.scala

```diff
@@ -53,6 +54,7 @@ class MarshallingDirectivesExamplesSpec extends RoutingSpec with CompileOnlySpec
 
   "example-entity-with-raw-json" in {
     // #example-entity-with-raw-json
+    import spray.json.JsValue
```

Comment:
Another suspicious change albeit this one seems harmless

Reply:
Part of 63b3437048, likely not strictly needed but in general we try to include relevant imports in code snippets that end up in the docs.